com.reprezen.swagedit.openapi3.tests/src/com/reprezen/swagedit/openapi3/assist/OpenApi3ContentAssistProcessorTest.xtend

```diff
@@ -139,16 +146,37 @@ class OpenApi3ContentAssistProcessorTest {
 
 		proposals = test.apply(processor, "2")
 		assertThat(proposals.map[(it as StyledCompletionProposal).replacementString], hasItems())
-		
+
 		proposals = test.apply(processor, "3")
 		assertThat(proposals.map[(it as StyledCompletionProposal).replacementString], hasItems())
-		
+
 		proposals = test.apply(processor, "4")
 		assertThat(proposals.map[(it as StyledCompletionProposal).replacementString], hasItems())
 
 		proposals = test.apply(processor, "5")
-		assertThat(proposals.map[(it as StyledCompletionProposal).replacementString],  
+		assertThat(
+			proposals.map[(it as StyledCompletionProposal).replacementString],
 			hasItems("int32", "int64", "float", "double", "byte", "binary", "date", "date-time", "password", "")
 		)
 	}
+
+// TODO
+//	@Test
+	def void testResponseStatusCode() {
```

Comment:
Our maven build runs tests as Eclipse plugin test. Let's re-enable this test so our CI could detect any regression.